pyogrio/_ogr.pxd

```diff
@@ -283,6 +283,14 @@ cdef extern from "ogr_api.h":
     int             OGR_F_GetFieldAsInteger(OGRFeatureH feature, int n)
     int64_t         OGR_F_GetFieldAsInteger64(OGRFeatureH feature, int n)
     const char*     OGR_F_GetFieldAsString(OGRFeatureH feature, int n)
+    char **         OGR_F_GetFieldAsStringList(OGRFeatureH feature, int n)
+    int *           OGR_F_GetFieldAsIntegerList(
+                        OGRFeatureH feature, int n, int* pnCount)
+    int64_t *       OGR_F_GetFieldAsInteger64List(
```

Comment:
Unfortunately, it looks like marking this as `int64_t` is failing on at some platforms:

```
error: assignment to ‘int64_t *’ {aka ‘long int *’} from incompatible pointer type ‘const GIntBig *’ {aka ‘const long long int *’} [-Wincompatible-pointer-types]
```

[error here](https://github.com/geopandas/pyogrio/actions/runs/17311694459/job/49146952701#step:5:805)

Unfortunately not caught by our tests, only when building the wheels.

Reply:
Yes, I saw it as well... I'm solving it...